sdk/storage/azure-storage-blob/azure/storage/blob/aio/_models.py

```diff
@@ -141,3 +141,39 @@ def _build_item(item):
             blob = FilteredBlob(name=item.name, container_name=item.container_name, tags=tags)
             return blob
         return item
+
+
+class PageRangePaged(AsyncPageIterator):
```

Comment:
I think these days we might not have to actually create this entire PageIterator classes.... let me see if I can find an example of a more simple implementation.